packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -65,22 +66,21 @@ export default function PostOptionsMenu({
 
     if (blockSource) {
       await unfollowSource({ source: reportedPost?.source });
-      await refreshFeed();
     }
   };
 
   const onBlockSource = async (): Promise<void> => {
     await unfollowSource({ source: post?.source });
     onMessage(`🚫 ${post?.source?.name} blocked`, postIndex);
     await new Promise((resolve) => setTimeout(resolve, 1000));
-    await refreshFeed();
+    onRemovePost?.(postIndex);
```

Comment:
This can go into a new function that gets the notification message and does all the rest

Reply:
This particular timeout is an extra for the message to show before removing it.
Do you want this to use the new hook?

Reply:
Did remove the promise here and await the onMessage as this is coming from the hook

Reply:
I meant an internal function, a helper function. Not a hook.
Something like:
```
async function foo (notification) {
onMessage(notification, postIndex);
    await new Promise((resolve) => setTimeout(resolve, 1000));
    onRemovePost?.(postIndex);
}